docs/developer-tools/runtime/services/marlowe-contract.md

```diff
@@ -5,19 +5,49 @@ title: Marlowe contract service
 The `marlowe-contract` executable provides services related to managing the Marlowe Runtime contract store.
 
 ```console
-marlowe-contract : a contract storage service for the Marlowe Runtime.
+marlowe-contract: Contract storage service for the Marlowe Runtime.
 
-Usage: marlowe-contract [-h|--host HOST_NAME] [-p|--port PORT_NUMBER]
-                        [--query-port PORT_NUMBER] [--transfer-port PORT_NUMBER]
-                        [-b|--buffer-size INTEGER] [-s|--store-dir DIR]
-                        [--store-staging-dir DIR]
+Usage: marlowe-contract [--version] [-h|--host HOST_NAME] 
+                        [-p|--port PORT_NUMBER] [--query-port PORT_NUMBER] 
+                        [--transfer-port PORT_NUMBER] 
+                        [--chain-sync-host HOST_NAME] 
+                        [--chain-sync-query-port PORT_NUMBER] 
+                        [--marlowe-sync-host HOST_NAME] 
+                        [--marlowe-bulk-port PORT_NUMBER] 
+                        [-b|--buffer-size INTEGER] [-s|--store-dir DIR] 
+                        [--store-staging-dir DIR] 
                         [--store-lock-microseconds-between-retries MICRO_SECONDS]
-                        [--http-port PORT_NUMBER]
+                        [--http-port PORT_NUMBER] [--min-contract-age MINUTES] 
+                        [--max-store-size BYTES]
 
-  Contract storage service for Marlowe Runtime
+  The contract storage service for the Marlowe Runtime. It manages a crucial component of the
+  Marlowe runtime: the contract store. The contract store is a content-addressable store of
+  contracts indexed by their hashes. Contracts can refer to sub-contracts via their hashes via
+  the MerkleizedCase construct. The contract store is used to store the continuations of a contract
+  after it has been merkleized (a process which recursively replaces Case constructs with MerkleizedCase
+  constructs). It is also used to lookup continuations when applying inputs to a merkleized contract.
+  This component exposes three protocols: marlowe load, marlowe transfer, and contract store query.
+
+  The marlowe load protocol is one way to import a contract incrementally into the store. It presents
+  a stack-based interface for pushing a contract depth-first into the store.
+
+  The marlowe transfer protocol is the other way to import a contract incrementally into the store.
+  It leverages the Marlowe object model to allow bundles of user-defined marlowe objects to be streamed
+  into the store. marlowe-contract will link the contract on-the-fly, merkleize the intermediate contracts,
+  and build the final contract incrementally. This protocol is generally more efficient and flexible than
+  Marlowe load because it allows duplicate sub-contracts to be pre-abstracted by the user. It also supports
+  An export mode which will stream the closure of a contract from the store to the client as a Marlowe object
```

Comment:
Suggest "An" should be "an"

"It also supports an export mode..."

Reply:
```suggestion
  an export mode which will stream the closure of a contract from the store to the client as a Marlowe object
```